### PR TITLE
Improve web consistency with native

### PR DESCRIPTION
**Pull request by arasrezaei**
- [x]  Added play(start,end) method.
- [x] Prevent animation from reseting after playing, just like native.
- [x]  Removed Lottie-files package because dotLottie package can handle all.
- [ ] you tell me

fixes https://github.com/lottie-react-native/lottie-react-native/issues/1195

### CODE REVIEW

**Comment by TheRogue76** [comment]
Beautiful. Feel free to ping for any info on anything, or when you want to mark this as ready for review.

File: packages/core/src/LottieView/index.web.tsx [hunk 1]
```diff
@@ -130,7 +181,7 @@ const LottieView = forwardRef(
         direction={direction}
       />
     );
-  },
+  }
 );
 
-export { LottieView };
+export default LottieView;
```
Comment (TheRogue76) [inline packages/core/src/LottieView/index.web.tsx:187]: Why turn it into a default import?

File: packages/core/src/LottieView/index.web.tsx [hunk 2]
```diff
@@ -25,17 +35,18 @@ const LottieView = forwardRef(
...
     }: LottieViewProps,
-    ref: any,
+    ref: any
   ) => {
     const sources = parsePossibleSources(source);
-    const isLottie =
-      sources.sourceName?.includes('.lottie') || !!sources.sourceDotLottieURI;
-    const lottieSource = sources.sourceDotLottieURI || sources.sourceName;
-    const jsonSource = sources.sourceURL || sources.sourceJson;
+    //const isLottie =  sources?.sourceName?.includes(".lottie") || !!sources?.sourceDotLottieURI;
```
Comment (TheRogue76) [inline packages/core/src/LottieView/index.web.tsx:41]: You can probably delete this

**Review by TheRogue76 — COMMENTED** [review]
Looking good code-wise. Please address my comments and we can get this ready for merge. I will do a test as well as soon as the comments are addressed, but much appreciated!

**Comment by TheRogue76** [comment]
@arasrezaei I can not merge your changes until you respond to the reviews I left. Please do not mark them as resolved until they are addressed. Thank you

**Comment by arasrezaei** [comment]
> @arasrezaei I can not merge your changes until you respond to the reviews I left. Please do not mark them as resolved until they are addressed. Thank you

now conflicts has been resolved, please review again

**Review by TheRogue76 — APPROVED** [review]
Much appreciated

**Pull request merged**